src/main/java/uk/gov/hmcts/reform/civil/handler/callback/user/SetAsideJudgmentCallbackHandler.java

```diff
@@ -41,6 +41,10 @@ public class SetAsideJudgmentCallbackHandler extends CallbackHandler {
     protected final ObjectMapper objectMapper;
     private final SetAsideJudgmentOnlineMapper setAsideJudgmentOnlineMapper;
     private static final String ERROR_MESSAGE_DATE_ORDER_MUST_BE_IN_PAST = "Date must be in the past";
+    private static final String ERROR_MESSAGE_APPLICATION_DATE =
+        "Application date to set aside judgment must be on or before the date of the order setting aside Judgment";
+    private static final String ERROR_MESSAGE_DEFENCE_DATE =
+        "Date the defence was received must be on or before the date of the order setting aside Judgment";
```

Comment:
Is the J suppose to be capital in last word in both strings?

Reply:
updated